docs/release-notes/2.41.0.md

```diff
@@ -0,0 +1,79 @@
+For a more streamlined and robust database deployment experience, try [Percona Everest](http://per.co.na/pmm-to-everest).
+
+## New Features
+
+- [PMM-8655](https://jira.percona.com/browse/PMM-8655) - PMM now captures both QAN and Victoria metrics data during network outages and delivers it from PMM Clients to PMM Server as soon as the connection is restored.
+- [PMM-12459](https://jira.percona.com/browse/PMM-12459) - The [pmm_dump client utility](https://docs.percona.com/pmm-dump-documentation/index.html) previously available as a standalone client for PMM Server is now readily accessible within the PMM user interface.
+
+## Improvements
+
+- [PMM-11341](https://jira.percona.com/browse/PMM-11341) - PMM 2.41.0 introduces limit for Auto-discovery in PostgreSQL, a feature that dynamically discovers all databases in your PostgreSQL instance. Limiting Auto-discovery reduces connections and prevents high CPU and RAM usage caused by multiple databases, thus optimizing performance.
+- [PMM-12375](https://jira.percona.com/browse/PMM-12375) - Starting with PMM 2.41.0, each instance of a service gets a `version` attribute in the PMM Inventory UI.
+- [PMM-12422](https://jira.percona.com/browse/PMM-12422) - PMM 2.41.0 introduces a new flag called `--expose-exporter`. When you enable this flag any IP address, either from a local system or from anywhere on the internet, can access exporter endpoints. If the flag is not enabled, the exporter will be available only locally.
+- [PMM-12544](https://jira.percona.com/browse/PMM-12544) - Added deprecation notices to the PMM documentation DBaaS pages. For a more streamlined and robust database deployment experience, try [Percona Everest](http://per.co.na/pmm-to-everest).
+- [PMM-12549](https://jira.percona.com/browse/PMM-12549) - Added support for the latest MongoDB version. You can now use PMM to monitor MongoDB 7 databases.
+
+
+## Components upgrade
+
+- [PMM-12154](https://jira.percona.com/browse/PMM-12154) - Updated postgres_exporter to version [0.14.0](https://github.com/prometheus-community/postgres_exporter/releases). With this update, we have resolved several performance issues and eliminated the creation of multiple connections per database.
+- [PMM-12223](https://jira.percona.com/browse/PMM-12223) - Clickhouse has been updated to version 23.8.2.7, which optimizes memory and CPU usage to improve system performance.
+
+
+## Bugs Fixed
+
+- [PMM-4712](https://jira.percona.com/browse/PMM-4712) - We have addressed the issue where the [pprof](https://github.com/google/pprof) heap reports for postgres_exporter were missing. 
+- [PMM-12626](https://jira.percona.com/browse/PMM-12626) - We updated the GO version used to build PMM to its latest version to mitigate potential vulnerabilities.
+- [PMM-12658](https://jira.percona.com/browse/PMM-12658) - Corrected a typo in the MongoDB cluster summary dashboard.
+
```

Comment:
```suggestion
- PMM-11714 - must in RN. We have some ADDITIONAL RELEASE NOTES TEXT for it . changing the LEvel - to public
- PMM-12414, PMM-12631
- PMM-12562 - need to be rechecked with QA maybe it's a defect for current version 
- PMM-12576
- PMM-12630
- PMM-12660

```

Reply:
based on versions, it's not a defect, as 2.37.1 were affected, not 2.41 
<img width="977" alt="image" src="https://github.com/percona/pmm-doc/assets/25495422/14e9c975-70ed-4128-8650-6c1cee85ca92">

Reply:
Added.